README.md

```diff
@@ -1,17 +1,175 @@
 # DELETE Forms and Requests
 
-## Objectives
+So far, we've worked with two pieces of the CRUD puzzle:
 
-- draw a delete route mapping to a destroy action
-- explain the problem with submitting a delete Requests
-- wrap a delete button in a form with _method=delete
-- use form_tag to build a delete form for an object
-- build a destroy action that finds the instance and destroys it and redirects to index
-- use form_for to build a delete form for an object
-- use link_to and button_to :method => :delete but explain briefly how the JS is working to do the same thing above
+- **C**REATING records, using HTTP `POST` requests
+- **R**EADING records, using HTTP `GET` requests.
 
-## Notes
+Two pieces remain:
 
-Goal is to get them to understand how the DELETE actions are built.
+- **U**PDATING records, using HTTP `PATCH` requests.
+- **D**ELETING records, using HTTP `DELETE` requests.
+
+But, all is not well in Browsertown. In many cases, sending a request with
+the `PATCH` or `DELETE` method will not work.
+
+Why? What can we do as a workaround?
+
+
+# Objectives
+
+After this lesson, you'll be able to...
+
+- Draw a `delete` route mapping to a `#destroy()` action
+- Explain the problem with submitting `delete` requests
+- Use `form_tag` to build a delete form for an object
+- Build a `#destroy()` action that finds the instance, destroys it, and
+  redirects to index
+- Use `link_to` and `button_to :method => :delete` to destroy an object without
+  a form.
+
+
+# Ignorance is bliss
+
+Before we dive into the problem with `PATCH` and `DELETE` requests, let's proceed
+as if we were none the wiser, and set up our route and form like usual:
+
+```ruby
+# config/routes.rb
+
+delete 'people/:id', to: 'people#destroy'
+```
+
+```erb
+# app/views/people/show.html.erb
+
+<h2><%= @person.name %></h2>
+<%= @person.email %>
+<%= form_tag people_path(@person.id), method: "delete" %>
+  <%= submit_tag "Delete #{@person.name}" %>
+<% end %>
+```
+
+But, wait a minute... there's something weird about the output we get:
+
+```html
+<h2>Caligula</h2>
+caligula@rome-circa-50-AD.com
+<form accept-charset="UTF-8" action="/people/1" method="post">
+  <input name="_method" type="hidden" value="delete" />
+  <input name="utf8" type="hidden" value="&#x2713;" />
+  <input name="authenticity_token" type="hidden" value="f755bb0ed134b76c432144748a6d4b7a7ddf2b71" />
+  <input name="commit" type="submit" value="Delete Caligula" />
+</form>
+```
+
+Enhance!
+
+> <form accept-charset="UTF-8" action="/people/1" method="**post**">
+>   <input name="_method" type="hidden" value="**delete**" />
```

Comment:
I like this exposition a lot. I like how you show the HTML and build up to this.